packages/mdc-select/foundation.js

```diff
@@ -85,14 +94,17 @@ export default class MDCSelectFoundation extends MDCFoundation {
     this.displayViaKeyboardHandler_ = (evt) => this.handleDisplayViaKeyboard_(evt);
     this.selectionHandler_ = ({detail}) => {
       const {index} = detail;
-      this.close_();
+      const removeFloatingLabelClass = false;
```

Comment:
Does this const exist purely for code style? It seems unnecessary since it's used only once in close proximity in each case.  If we do keep it, I'd suggest renaming to `shouldRemoveFloatingLabelClass`.